openapi-core/src/main/java/io/ballerina/openapi/core/generators/service/model/OASServiceMetadata.java

```diff
@@ -67,8 +69,7 @@ public static class Builder {
         private Filter filters;
         private boolean nullable = false;
 
-        private String license = "// AUTO-GENERATED FILE. DO NOT MODIFY.\n\n" +
-                "// This file is auto-generated by the Ballerina OpenAPI tool.\n";
+        private String license = DEFAULT_FILE_HEADER;
```

Comment:
Do need to have this `license` variables? can't we directly use the `DO_NOT_MODIFY_FILE_HEADER` instead?